theano/gof/opt.py

```diff
@@ -2669,7 +2674,9 @@ def check_stack_trace(f_or_fgraph, ops_to_check='last', bug_print='raise'):
     # if ops_to_check is a list/tuple of ops
     elif isinstance(ops_to_check, (tuple, list)):
         apply_nodes_to_check = [node for node in fgraph.apply_nodes
-                                if node.op in ops_to_check]
+                                if node.op in ops_to_check or
+                                isinstance(node.op, ops_to_check) or
+                                isinstance(node.op.scalar_op, ops_to_check)]
```

Comment:
Fixed, for cases like that https://github.com/Theano/Theano/pull/4376/commits/d4b12260e73cf7001ebae258d2e80bb10331884b#diff-dbe5c9d6bf5794a9f19f73ec4754678bR369